package.json

```diff
@@ -473,15 +473,10 @@
 		"@types/vscode": "^1.31.0",
 		"asciidoctor-kroki": "^0.8.2",
 		"asciidoctor-plantuml": "1.5.0",
-		"copy-paste": "^1.3.0",
 		"file-url": "^3.0.0",
 		"follow-redirects": "^1.10.0",
-		"fs-extra": "^8.1.0",
-		"https-proxy-agent": "^4.0.0",
```

Comment:
>  ERROR in ./src/commands/exportAsPDF.ts
> Module not found: Error: Can't resolve 'https-proxy-agent' in '/home/runner/work/asciidoctor-vscode/asciidoctor-vscode/src/commands'
>  @ ./src/commands/exportAsPDF.ts 22:22-50
>  @ ./src/commands/index.ts
>  @ ./src/extension.ts
> 
> 

I see that `https-proxy-agent` is used in the pdf export from the CI output.

Reply:
thanks for noticing it